packages/client/hmi-client/src/components/workflow/tera-stratify.vue

```diff
@@ -130,17 +131,52 @@
 				</div>
 			</section>
 		</section>
-		<section class="step-1" v-else-if="stratifyView === StratifyView.Output">
+		<section class="output" v-else-if="stratifyView === StratifyView.Output">
 			<div>If this is not what you expected, go back to the input page to make changes.</div>
 			<Accordion multiple :active-index="[0, 1]">
 				<AccordionTab header="Stratified model">
-					<div class="step-1-inner"></div>
+					<tera-stratify-output-model-diagram v-if="stratifiedModel" :model="stratifiedModel" />
 				</AccordionTab>
 				<AccordionTab header="Strata model">
-					<div class="step-1-inner"></div>
+					<tera-strata-model-diagram
+						v-if="typedStrataModel"
+						:strata-model="typedStrataModel"
+						:show-reflexives-toolbar="false"
+					/>
 				</AccordionTab>
 			</Accordion>
-			<div>Saved as: {{ stratify_output.name }}</div>
+			<div>
+				Saved as:
+				<Button
+					class=".p-button-link"
+					:label="stratifiedModel?.name"
+					icon="pi pi-pencil"
+					iconPos="right"
+					text
+					@click="
+						emit('open-asset', {
+							assetName: `${stratifiedModel?.name}`,
+							pageType: ProjectAssetTypes.MODELS,
+							assetId: stratifiedModel?.id
+						})
+					"
+				/>
```

Comment:
I don't understand, are we:
- saving it as a new model? or
- opening the new model in a new tabl?

Reply:
Then the title/label should not say `Save as` - basically this is false advertising.